### PR TITLE
Update cal-com-share-meeting-links extension

### PR DESCRIPTION
**Pull request by AlexIsMaking**
## Description

Displays prices for events.

CleanShot 2024-07-26 at 15 10 47@2x

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/cal-com-share-meeting-links/src/api/cal.com.ts [hunk 1]
```diff
@@ -123,7 +123,7 @@ export interface updateBookingContent {
 }
 
 const defaultBaseUrl = "https://api.cal.com/v1/";
-const { token } = getPreferenceValues<Preferences>();
+const { token } = getPreferenceValues<PreferenceValues>();
```
Comment (jfkisafk) [inline extensions/cal-com-share-meeting-links/src/api/cal.com.ts:126]: Nitpick (non-blocking): Why are we changing this? I thought raycast recommends using the types generated in _raycast-env.d.ts_ at the end of [this doc](https://developers.raycast.com/api-reference/preferences#getpreferencevalues).

File: extensions/cal-com-share-meeting-links/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Cal.com Share Meeting Links Changelog
 
+## [Update] - 2024-07-26
```
Comment (jfkisafk) [inline extensions/cal-com-share-meeting-links/CHANGELOG.md:3]:
```suggestion
## [Update] - {PR_MERGE_DATE}
```

**Review by jfkisafk — APPROVED** [review]
Overall LGTM, just minor comments/questions. Thanks for updating the screenshot.

File: extensions/cal-com-share-meeting-links/CHANGELOG.md (hunk 2 above)
Comment (AlexIsMaking) [reply extensions/cal-com-share-meeting-links/CHANGELOG.md]: Thanks for the suggestions. My compiler wasn't liking the Preferences type reference earlier but it looks fine now.

**Review by jfkisafk — APPROVED** [review]

**Comment by pernielsentikaer** [comment]
Thanks for the update 🔥

Regarding the new screenshot, do you think it would be possible to take it with the same default dark theme as the others? 🙂

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**